.circleci/config.yml

```
@@ -43,17 +43,15 @@ executors:
     parameters:
       node-version:
         # renovate datasource=node-version depName=node
-        default: 20.18.2
-        type: string
+        default: 22.13.0        type: string
```

je pense que le yaml n'est plus valide...
```suggestion
        default: 22.13.0
        type: string
```